projects/utils-maps/src/lib/ol/ol.ts

```diff
@@ -0,0 +1,184 @@
+import { Map, View } from 'ol';
```

Comment:
I would not name this file ol.ts because this can maybe bring confusion that it contains the openlayers library.
Use something like **ol-utils** or **utils-ol** or **ukis-ol-utils**.